src/components/Complete/index.js

```diff
@@ -12,9 +12,9 @@ class Complete extends Component {
   render ({message, submessage}) {
     return (
       <div>
-        <div className={theme.step}>
           <span className={`${theme.icon}  ${style.icon}`}></span>
           <h1 className={theme.title}>{message}</h1>
+        <div className={theme.textWrapper}>
```

Comment:
does text wrapper have some meaning behind it? is it for any text?

Reply:
It's probably a bad name - it's usually used for text. Really the uploaderWrapper is thin and the textWrapper is wide. I will rename them accordingly - `thinWrapper` `fatWrapper`

Reply:
read your description. I wonder if there is a better name for it, considering is not just text we are wrapping.

Reply:
@danielspagnolo thoughts on it?